app/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+DS_Store
```

Comment:
This should be `.DS_Store`. You'll need to add that, and remove the DS_Store you've added.

Reply:
Done

Reply:
Still seeing it?

Reply:
Done locally. I haven't pushed yet, will do shortly

Reply:
Cool. In the future, if it hasn't been pushed and solved in this PR, then it isn't done.